Enterprise-CMCS/macpro-mdct-mfp
# knock out these aws-sdk upgrades

### Description
Takin' care of snyk in one PR


### Related ticket(s)
N/A

---
### How to test

Log in to MFP and make sure nothing is broken? That report data is saved to the db.
